core/build.gradle

```diff
@@ -44,13 +44,17 @@ dependencies {
     compile 'com.graphql-java-kickstart:graphql-java-tools:5.5.1'
     compile 'com.google.cloud:google-cloud-storage:1.85.0'
     compile 'com.github.cloudyrock.mongock:mongock-core:3.0.0.BETA'
+    compile 'io.micronaut.configuration:micronaut-security-oauth2:1.2.3.BUILD-SNAPSHOT'
```

Comment:
Travis doesn't build because of:
```
Could not find io.micronaut.configuration:micronaut-security-oauth2:1.2.3.BUILD-SNAPSHOT
```
Maybe we shouldn't use SNAPSHOT version?